wiki/Best_Of/en.md

```diff
@@ -7,277 +6,314 @@ Index page of all the "Best Of" series.
 
 #### 2009 osu!standard
 
-| Placing | 2009 Results                                                            |
-|---------|-------------------------------------------------------------------------|
-| \#1     | [Queen - Don't Stop Me Now (m980)](https://osu.ppy.sh/s/5381)                 |
-| \#2     | [Yoko Hikasa - Don't Say "Lazy" (Dangaard)](https://osu.ppy.sh/s/6257)        |
+| Placing | 2009 Results |
+|---------|--------------|
+| \#1     | [Queen - Don't Stop Me Now (m980)](https://osu.ppy.sh/s/5381)  |
+| \#2     | [Yoko Hikasa - Don't Say "Lazy" (Dangaard)](https://osu.ppy.sh/s/6257)  |
 | \#3     | [Toyosaki Aki - Cagayake! GIRLS (TV Size) (DJPop)](https://osu.ppy.sh/s/6358) |
 
--    [Top 30](https://osu.ppy.sh/forum/t/22162)
--    [Charts](https://osu.ppy.sh/p/chart?ch=best2009)
--    Download:
-     -    Mediafire
-          -    [Part 1](https://www.mediafire.com/?plr6t9co71818fl)
-          -    [Part 2](https://www.mediafire.com/?d1bi0b447zrb70r)
-     -    osu!mirror
-          -    [Part 1](https://osu.yas-online.net/tagthis.php?tag=R20)
-          -    [Part 2](https://osu.yas-online.net/tagthis.php?tag=R21)
--    [News](https://osu.ppy.sh/forum/viewtopic.php?f=9&amp;t=21059)
--    [Results](https://osu.ppy.sh/forum/t/22162)
+- [Top 30](https://osu.ppy.sh/forum/t/22162)
+- [Charts](https://osu.ppy.sh/p/chart?ch=best2009)
+- Download:
+  - Mediafire
```

Comment:
I believe we can remove the mediafire level from the lists as ots the only point


just remove the mediafire .-. 


same with everything else